.evergreen/run-mongohouse-local.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+#
+# This script builds and launches a local mongohoused for testing.
+#
+# There is no corresponding 'shutdown' script; this project relies
+# on Evergreen to terminate processes and clean up when tasks end.
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+ORIG_DIR="$(pwd)"
+
+# Configure git to use the git protocol
+git config --global url."git@github.com:".insteadof "https://github.com/"
```

Comment:
I didn't see this done in any other files in the repository. Is this necessary, given that `git clone` below is using the SSH address?

`git config --global` gave me pause here, since that could potentially change the system configuration for anything that executes after this script.

Reply:
This is necessary because one of the dependencies, `mongoast`, is a private repository and the go tool, during build, can't resolve a private repository without this bit.

Reply:
Thanks for context. @jstewart-mongo: can you summarize the above justification into a comment for the benefit of future maintainers that come across this file?